htdocs/js/jquery.dynamictable.js

```diff
@@ -72,21 +76,58 @@
         }
 
 
-    };
+    }, wrapTable = function (table) {
+        $(table).wrap("<div class=\"row\"></div>");
 
-    $.fn.DynamicTable = function () {
-        this.filter("table").each(function () {
-            var leftLink, rightLink, table = this;
-            // Add wrapper code necessary for bootstrap carousel
-            $(this).wrap("<div class=\"carousel slide\" data-ride=\"carousel\"></div>");
-            $(this).wrap("<div class=\"carousel-inner\"></div>");
+        // Add wrapper code necessary for bootstrap carousel
+        $(table).wrap("<div class=\"carousel slide col-xs-12\"></div>");
 
-            // Add wrapper necessary for dynamictable code
-            $(this).wrap("<div class=\"dynamicContentWrapper table-scroll\" style=\"overflow-x: auto\"></div>");
+        // Add wrapper necessary for dynamictable code
+        $(table).wrap("<div class=\"dynamicContentWrapper table-scroll\" style=\"overflow-x: auto\"></div>");
 
-            // Add links for carousel
-            $(this).after('<a class="left carousel-control" href="#"><span class="glyphicon glyphicon-chevron-left"></span></a><a class="right carousel-control" href="#" data-slide="next"><span class="glyphicon glyphicon-chevron-right"></span></a>');
+        // Add links for carousel
+        $(table).after('<a class="left carousel-control" href="#"><span class="glyphicon glyphicon-chevron-left"></span></a><a class="right carousel-control" href="#" data-slide="next"><span class="glyphicon glyphicon-chevron-right"></span></a>');
+    }, freezeColm = function (tableID, colm_static) {
+        var statColPos = $("." + tableID).offset().left,
```

Comment:
the tableID is used to identify the column that is to be frozen, not for the table itself. Would it be better to call the class identifier something that is more descriptive to the column and not the table?